web/app/scripts/src/libraries.es.js

```diff
@@ -0,0 +1 @@
+export const angular = window.angular;
```

Comment:
I see this is a wrapper to expose angular as an ES6 module which is nice. It's a shame I had hoped angular itself would expose an ES6 module hook that you can point directly at, but it doesn't look like it :(